### ISSUE TITLE
Is it possible to modify the HTTP headers for CORS ? 

### ISSUE THREAD

**Issue by icaraps** [adobe/helix-pages#152]
Here's the use case: 
* 2 repos A and B up and running via helix
* repo A tries to fetch content from repo B e.g with `window.fetch()`

Currently it fails due to CORS policy: 
```
Access to fetch at 'https://repoA.project-helix.page' from origin 'https://repoB.project-helix.page' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
```
cc @kptdobe

**Comment by trieloff** [comment]
We could (but shouldn’t) loosen the CORS headers in the helix-pages VCL.

Can you tell us more about the specifics of the request you are making, so that we can discuss implementation or workarounds?

**Comment by icaraps** [comment]
@trieloff here's the project https://github.com/icaraps/codelabs see https://codelabs-icaraps.project-helix.page/lab.html?src=/labs/faast-path-to-beautiful-apps

Basically, I'd like to show labs content but I don't want to "own" the content and I don't want to impose a helix structure to labs (e.g default helix header etc). 

So currently repoA (=codelabs) and repoB (=lab) are merged into a single repo but with CORS for labs I could pull in lab content from outside of the codelabs repo and manipulate the content as I need it. 

Does it make sense ? cc @kptdobe

**Comment by rofe** [comment]
@icaraps Helix Pages doesn't support separate repos for code and content, you'd need to [deploy your own Helix project](https://github.com/adobe/helix-home/blob/master/getting-started.md) with a `helix-config.yaml` for that...

You could keep your `.md` files e.g. in a Google Drive or OneDrive folder. See https://github.com/adobe/helix-pages/issues/67 or https://github.com/adobe/helix-word2md for instructions how to do create and map such a shared folder. An project using both: https://github.com/davidnuescheler/theblog/blob/master/fstab.json

**Comment by trieloff** [comment]
As @rofe said, using Helix proper (with your own runtime account and Fastly service) is probably the best choice for your use case.

We should not add unrestricted CORS headers to helix-pages, so that content owners remain in control over the usage and appearance of their content, so I'll close this issue, as the initial question has been answered.
